ui/src/app/views/research-purpose.tsx

```diff
@@ -0,0 +1,153 @@
+import {Component} from '@angular/core';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+import {Button, Clickable} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {TwoColPaddedTable} from 'app/components/tables';
+import {EditComponentReact} from 'app/icons/edit';
+import {workspacesApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {reactStyles, ReactWrapperBase, withCurrentWorkspace, withUserProfile} from 'app/utils';
+import {sliceByHalfLength} from 'app/utils/index';
+import {navigate} from 'app/utils/navigation';
+import {WorkspaceData} from 'app/utils/workspace-data';
+import {WorkspacePermissions} from 'app/utils/workspace-permissions';
+import {Authority, Profile, SpecificPopulationEnum} from 'generated/fetch';
+import {ResearchPurposeDescription, ResearchPurposeItems, specificPopulations} from './workspace-edit';
+
+const styles = reactStyles({
+  mainHeader: {
+    fontSize: '20px', fontWeight: 600, color: colors.primary, marginBottom: '0.5rem',
+    display: 'flex', flexDirection: 'row'
+  },
+  sectionHeader: {
+    fontSize: '16px', fontWeight: 600, color: colors.primary, marginTop: '1rem'
+  },
+  sectionText: {
+    fontSize: '14px', lineHeight: '24px', color: colors.primary, marginTop: '0.3rem'
+  }
+});
+
+
+export const ResearchPurpose = fp.flow(withCurrentWorkspace(), withUserProfile())(
+  class extends React.Component<
+    {profileState: { profile: Profile, reload: Function }, workspace: WorkspaceData},
+    {publishing: boolean, workspacePermissions: WorkspacePermissions}> {
```

Comment:
Semantically, I wouldn't expect publishing to be owned by the Research Purpose component. Should this be moved into the WorkspaceAbout component?

Reply:
yeah good thought.  although this is not user-facing yet and is a pretty rough sketch